content/docs/services/relational-database.md

```diff
@@ -149,7 +146,11 @@ dump.
 $ pg_restore --clean --no-owner --no-acl --dbname={database name} backup.pg
 ```
 
-### Rotating credentials
+## Encryption
+
+Every RDS instance configured through cloud.gov is [encrypted at rest](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.Encryption.html).
```

Comment:
Do you think we should link to the place in our broker config where this happens, @brittag? So that customers can use that in the course of gathering "evidence" for their auditors that we do in fact do this?